.github/workflows/deploy.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install mdbook
+      run: |
+        mkdir mdbook
+        curl -Lf https://github.com/rust-lang/mdBook/releases/download/v0.4.8/mdbook-v0.4.8-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=./mdbook
+        echo `pwd`/mdbook >> $GITHUB_PATH
+    - name: Generate Book
+      run: |
+        ./generate-book.sh
+    - name: Deploy GitHub Pages
+      uses: rust-lang/simpleinfra/github-actions/static-websites@master
```

Comment:
FWIW: this action hasn't been maintained these days and [can be slow](https://github.com/rust-lang/simpleinfra/pull/61#issuecomment-754077884). Though I think it's the simplest way and enough here.

Reply:
Ah, I didn't know it wasn't recommended.  I went ahead and updated it to use a trivial script I've used elsewhere that doesn't need to deal with tokens or anything.